grafana_plugins_init/TAG

```diff
@@ -1 +1 @@
-0.0.3.1
+0.0.3.2
```

Comment:
Does this TAG change necessary?

Reply:
@binoue 
Yes.
The grafana_plugins_init container image uses the Ubuntu image as its base, and we need to reflect the update of the base.
cf. https://github.com/cybozu/neco-containers/blob/update-grafana-operator-3.10.1/grafana_plugins_init/Dockerfile